src/panels/config/zha/zha-node.ts

```diff
@@ -186,50 +178,27 @@ export class ZHANode extends LitElement {
   private _selectedNodeChanged(event: ItemSelectedEvent): void {
     this._selectedNodeIndex = event!.target!.selected;
     this._selectedNode = this._nodes[this._selectedNodeIndex];
-    this._selectedEntity = undefined;
     fireEvent(this, "zha-node-selected", { node: this._selectedNode });
     this._serviceData = this._computeNodeServiceData();
   }
 
   private async _onReconfigureNodeClick(): Promise<void> {
     if (this.hass) {
-      await reconfigureNode(this.hass, this._selectedNode!.attributes.ieee);
+      await reconfigureNode(this.hass, this._selectedNode!.ieee);
     }
   }
 
-  private _showNodeInformation(): void {
-    fireEvent(this, "hass-more-info", {
-      entityId: this._selectedNode!.entity_id,
-    });
-  }
-
   private _computeNodeServiceData(): NodeServiceData {
     return {
-      ieee_address: this._selectedNode!.attributes.ieee,
+      ieee_address: this._selectedNode!.ieee,
     };
   }
 
-  private _computeSelectCaption(stateObj: HassEntity): string {
-    return (
-      computeStateName(stateObj) + " (Node:" + stateObj.attributes.ieee + ")"
-    );
-  }
-
-  private _computeNodes(hass?: HomeAssistant): HassEntity[] {
-    if (hass) {
-      return Object.keys(hass.states)
-        .map((key) => hass.states[key])
-        .filter((ent) => ent.entity_id.match("zha[.]"))
-        .sort(sortByName);
-    } else {
-      return [];
-    }
-  }
-
-  private _onEntitySelected(
-    entitySelectedEvent: HASSDomEvent<ZHAEntitySelectedParams>
-  ): void {
-    this._selectedEntity = entitySelectedEvent.detail.entity;
+  private async _fetchDevices() {
+    this._nodes = await fetchDevices(this.hass!);
```

Comment:
Assigning twice _feels_ a bit weird, I would prefer:

```ts
    this._nodes = (await fetchDevices(this.hass!)).sort((a, b) => { … });
```

However, it's more like a nitpick, ignore if you want.